algebra-testing/src/main/java/com/hubspot/assertj/algebra/error/ResultShouldBeErrWithValue.java

```diff
@@ -10,11 +10,13 @@ private ResultShouldBeErrWithValue(String format, Object... arguments) {
     super(format, arguments);
   }
 
-  public static <T, E> ErrorMessageFactory shouldBeOkWithValue(Result<T, E> actual, Object value) {
+  public static <T, E> ErrorMessageFactory shouldBeErrWithValue(Result<T, E> actual, Object error) {
     if (actual.isErr()) {
-      return new ResultShouldBeErrWithValue("Expecting Result to be Err containing <%s> but contained <%s>", value, actual.unwrapErrOrElseThrow());
+      E actualError = actual.unwrapErrOrElseThrow();
+      return new ResultShouldBeErrWithValue("Expecting Result to be Err containing <%s> but contained <%s>. expected:<%s> but was:<%s>", error, actualError, error, actualError);
```

Comment:
this is all this PR is doing, appending the `expected:<%s> but was:<%s>` to the error message.